validation-test/stdlib/ArrayNew.swift.gyb

```diff
@@ -1610,9 +1610,8 @@ final class EvilCollection : CollectionType {
 
 for (step, evilBoundsCheck) in [ (1, true), (-1, false), (-1, true) ] {
 
-  let message = step < 0 && evilBoundsCheck
-    ? evilBoundsError
-    : "invalid CollectionType: count differed in successive traversals"
```

Comment:
Usage of the new `expectStableCollection` function ensures that evil collections get consistent fatal error messages.